README.rst

```diff
@@ -23,7 +23,7 @@ If you'd like to join them, please consider `sponsoring Twisted's <https://docs.
 What is this?
 -------------
 
-Twisted is an event-based framework for internet applications, supporting Python 3.6+.
+Twisted is an event-based framework for internet applications, supporting Python 3.8+.
```

Comment:
Maybe something like this 

```suggestion
Twisted is an Python based event-based framework for internet applications.
```

Reply:
"Python based event-based" is ... ugh, clunky.  I prefer @altendky's suggestion in https://github.com/twisted/twisted/pull/12354.  @adiroiban, are you OK with that?

Reply:
Or maybe:

```suggestion
Twisted is a Python event-based framework for internet applications.
```

It's probably worth having the word "Python" appear at least once in the README, and currently this is the only place it appears.

Reply:
Sure. No problem. We can go with `Twisted is an event-based framework for internet applications.`